terminus/tests/road_test.py

```diff
@@ -66,3 +68,22 @@ def test_length_empty(self):
     def test_length_empty_road(self):
         road = Street.from_points([])
         self.assertEqual(road.length(), 0.0)
+
+        '''
+        this test couldn't be implemented yet:
```

Comment:
This is what happens:
ERROR: test_node_bounding_boxes (road_test.RoadTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/jazmin/terminus/terminus/tests/road_test.py", line 79, in test_node_bounding_boxes
    road._add_node(node_1)
  File "/home/jazmin/terminus/terminus/models/road.py", line 133, in _add_node
    node.added_to(self)
  File "/home/jazmin/terminus/terminus/models/road_node.py", line 23, in added_to
    raise NotImplementedError()
NotImplementedError


is this 'NotImplementedError' something to be changed by some code in the future?
I wanted to create a road to test the method, but my problem was creating the road.